src/push.h

```diff
@@ -83,7 +83,7 @@ int git_push_add_refspec(git_push *push, const char *refspec);
  * Update remote tips after a push
  *
  * @param push The push object
- * @param signature The identity to use when updating reflogs
+ * @param callbacks The identity to use when updating reflogs
```

Comment:
This fixes it syntactically, but this isn't a copy-paste error; it's documentation for an old parameter. The whole line should be replaced by something relevant to the new function signature.